src/main/java/se/sundsvall/esigning/api/ProcessResource.java

```diff
@@ -48,9 +50,10 @@ public ProcessResource(ProcessService service) {
 	@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	public ResponseEntity<StartResponse> startProcess(
+		@Parameter(name = "municipalityId", description = "Municipality ID", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@RequestBody @NotNull @Valid SigningRequest request) {
 
-		final var startProcessResponse = new StartResponse(service.startProcess(request));
+		final var startProcessResponse = new StartResponse(service.startProcess(municipalityId, request));
 		LOGGER.info("Request for start of e-signing process for request {} has been received, resulting in an instance with id {}", request, startProcessResponse.getProcessId());
```

Comment:
Add municipalityId in this log entry?

Reply:
I did something similar in another application, but SonarCloud is not a fan of logging user input.

Reply:
I have now added it.